src/sugar3/graphics/objectchooser.py

```diff
@@ -170,6 +171,13 @@ def __init__(self, parent=None, what_filter=None, filter_type=None,
         self._filter_type = filter_type
 
     def run(self):
+        """
+        Runs the object chooser and displays it.
+        
+        Returns:
+            Gtk.ResponseType constant, the response received from displaying the
+                object chooser.
```

Comment:
What response types does it return?  What do they mean in the context of the chooser?